Sources/Deboogger.swift

```diff
@@ -47,34 +47,61 @@ public final class Deboogger {
         return window
     }()
 
-    private weak var pluginViewController: PluginViewController?
+    weak var pluginViewController: PluginViewController?
     public var viewController: UIViewController? {
         return pluginViewController
     }
     public var navigationController: UINavigationController? {
         return pluginViewController?.navigationController
     }
 
+    private var configuration: Configuration?
+    private var isCurrentlyShowing: Bool = false
+
+    private var _shouldShowAssistiveButton: Bool = false
+    var shouldShowAssistiveButton: Bool {
+        get {
+            #if targetEnvironment(simulator)
+                return true
```

Comment:
Maybe we should still have the ability to disable assistive button on simulator? It would be useful in case we'd actually start writing UI tests w/ screenshots.

Reply:
@rosberry, sounds reasonable, but unfortunately I have no idea how to implement this. Maybe we can pass some arguments on start for that case?

Reply:
@rosberry, Also this can be done as a separate update, I suppose

Reply:
> Maybe we can pass some arguments on start for that case?

Yes, that was my idea as well.
It'll be fine as a separate update.